mPower2/mPower2/SignInTaskViewController.swift

```diff
@@ -54,6 +55,17 @@ class SignInTaskViewController: RSDTaskViewController, SignInDelegate {
     var regionCode: String? = "US" // TODO: emm 2018-04-25 Handle non-US phone numbers for international studies
 
     init() {
+        if #available(iOS 12.0, *) {
```

Comment:
This file change looks good.  Curious, how do you test this for Netherlands?

Reply:
I have not found any way to do so, short of trying it on a NL phone or getting a SIM card from a NL carrier and swapping it in to your phone. I tested it for US and am basically going on faith that the iOS API works as described in other countries.